pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -192,6 +200,8 @@ func TestOpenShiftClusterStaticValidateProperties(t *testing.T) {
 			},
 			wantErr: "400: InvalidParameter: properties.provisioningState: The provided provisioning state 'invalid' is invalid.",
 		},
+	}
+	createTests := append(commonTests, []*validateTest{
```

Comment:
Here we check validation of worker profiles (and below similarly we test version) only on create. However we make sure that it's not possible to change the worker profiles, version and other fields on update in `TestOpenShiftClusterStaticValidateDelta`.